src/components/reader/pager/DoublePagedPager.tsx

```diff
@@ -100,7 +93,8 @@ export default function DoublePagedPager(props: IReaderProps) {
     }
 
     function nextPage() {
-        if (curPage < pages.length - 1) {
+        // Only go to next page if last page isn't displayed
+        if (curPage < pages.length - pagesDisplayed.current) {
```

Comment:
this will cause the chapter not to get marked as read anymore, since for that, the last page has to be set as the current page